node/src/service.rs

```diff
@@ -831,26 +874,26 @@ pub async fn new_full<
         let backend = backend.clone();
 
         move |deny_unsafe, subscription_executor: sc_rpc::SubscriptionTaskExecutor| {
-            let shared_voter_state = sc_consensus_grandpa::SharedVoterState::empty();
             let deps = crate::rpc::FullDeps {
                 client: client.clone(),
                 pool: pool.clone(),
                 select_chain: select_chain.clone(),
+                chain_spec: chain_spec.cloned_box(),
                 deny_unsafe,
                 command_sink: None,
                 eth: eth_rpc_params.clone(),
-                babe: crate::rpc::BabeDeps {
```

Comment:
This one was re-import or?

Reply:
Yes. I deleted our custom BabeDeps, BeefyDeps, and GrandpaDeps, and pulled them from polkadot_rpc. We don’t have any customizations for these structs here.